examples/create-new-meeting/Ballerina.toml

```
@@ -0,0 +1,15 @@
+[package]
+org = "laavanja"
```

Use the `wso2` organization for the examples.

```suggestion
org = "wso2"
```